src/components/MapInput/index.js

```diff
@@ -115,10 +116,18 @@ const MapInput = ({ className, value, onChange, mapOptions, ...otherProps }) =>
   }, [marker, location, hasLocation]);
 
   useEffect(() => {
-    if (!value?.geometrie?.coordinates && !value?.addressText) return;
+    if (!map) return;
+    if (!value?.location && !value?.addressText) return;
+
+    if (value?.location?.lat > 0) {
+      if (!isEqual(value?.location, state.location)) {
+        const currentZoom = map.getZoom();
+        map.flyTo(value.location, currentZoom);
+      }
```

Comment:
This clause still doesn't seem to be properly covered by a test case.

Reply:
Done